Tools/px_uploader.py

```diff
@@ -726,29 +726,6 @@ def main():
         print("WARNING: You should uninstall ModemManager as it conflicts with any non-modem serial device (like Pixhawk)")
         print("==========================================================================================================")
 
-    # We need to check for pyserial because the import itself doesn't
-    # seem to fail, at least not on macOS.
-    pyserial_installed = False
-    try:
-        if serial.__version__:
-            pyserial_installed = True
-    except:
-        pass
-
-    try:
-        if serial.VERSION:
-            pyserial_installed = True
-    except:
-        pass
-
-    if not pyserial_installed:
-        print("Error: pyserial not installed!")
-        print("")
-        print("You may need to install it using:")
-        print("    pip3 install --user pyserial")
-        print("")
-        sys.exit(1)
```

Comment:
I think you should still leave the import error like we have it in other places:

https://github.com/PX4/PX4-Autopilot/blob/eee08601affba7ae960b1703d489c06c4745505f/Tools/serial/generate_config.py#L10-L18

Reply:
Added.

We should be careful with these suggestions as it can be out of sync with the actual python executable or environment used.